aten/src/ATen/native/Pooling.cpp

```diff
@@ -132,6 +133,11 @@ Tensor max_pool2d(
     return at::mkldnn_max_pool2d(
         self, kernel_size, stride, padding, dilation, ceil_mode);
   }
+  if(xnnpack::use_max_pool2d(self, kernel_size, padding, stride,
```

Comment:
Why are we directly exposing this here? I thought, based on conv and linear, our philosophy was going to be not exposing this directly but require explicit opt-in via "xnnpackify.." short of transform of the network. @dreiss for comment.

Reply:
Also note that, this will also make it a little harder to fuse maxpool + relu.

Reply:
Binding it directly is good as it directly benefits existing models. It doesn't preclude us from doing more optimizations also like fusion you described.

Reply:
I agree.  With Dima.  We can have a separate diff that exposes the fused version.

Reply:
Great.